src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 10px;
+  line-height: 18px;
+  box-sizing: border-box;
+  color: #060b35;
+}
+
+.card {
+  background-color: $main-color;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    padding: 33px 20px 0;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__descript {
+    height: 36px;
+    width: 166px;
+    padding: 40px 16px 0;
+    margin-bottom: 16px;
+  }
+
+  &__descript-name {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__descript-code {
+    padding-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    padding-top: 24px;
+    justify-content: space-between;
+    width: 166px;
+    height: 18px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+    margin: 0;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__buy {
+    display: block;
+    padding: 16px;
+    width: 166px;
+  }
+
+  &__buy-link {
+    display: inline-block;
+    text-align: center;
+    width: 100%;
+    height: 40px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    color: $main-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    border: none;
+    text-decoration: none;
+  }
+
+  &__buy-link:hover {
+    background-color: $main-color;
+    border: 1px solid#00ACDC;
```

Comment:
There is a missing space between 'solid' and '#00ACDC'. It should be 'solid #00ACDC'. This is a minor code style issue.